assets/js/components/includes/DropdownFilter.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <select @change="getIndex(items)">
-  <option>
-  {{title}}
-  </option>
-    <option :selected="item.id == getFilterValue()"
```

Comment:
it is not smart to run a function for each element, since that returns a static value. use vue's computed